docs/ops/doc/Purpose.rst

```diff
@@ -1,50 +1,68 @@
-# Purpose
+=======
+Purpose
+=======
```

Comment:
Super Bikeshed Man wonders: Are we changing all headers from `#` style over to `=====`/`------` now? Or does `#` style not work in .rst? Either way: should the .md files not then be consistent with this style?

Reply:
Yeah, I'm pretty sure `#` means something different within rst [sections](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections)